src/routes/__root.tsx

```diff
@@ -74,8 +74,9 @@ function RootRoute() {
       <Toaster
         theme={theme as "light" | "dark" | "system"}
         offset={16}
+        mobileOffset={{ bottom: "60px" }}
```

Comment:
```suggestion
        mobileOffset={{ bottom: "68px" }}
```
A little bit of extra space so the margins are the same!